packages/cli/smoketest.sh

```diff
@@ -23,6 +23,7 @@ cd "${TESTDIR}"
 "./${BIN_NAME}" depends --appmap-dir ruby
 "./${BIN_NAME}" inventory --appmap-dir ruby
 "./${BIN_NAME}" openapi -d ruby -o /dev/null
+"./${BIN_NAME}" sequence-diagram -d ruby revoke_api_key.appmap.json
```

Comment:
This is a good idea, but it was actually `@appland/appmap install` that failed in the wild last time.

Reply:
Yeah, I was just playing around to see if I could provoke a failure.